### PR TITLE
fix the issue: outdated_tls_policy

### PR DESCRIPTION
**Pull request by SanaaYousaf**

### CODE REVIEW

**Comment by SanaaYousaf** [comment]
this commit fixes #3007 https://github.com/aquasecurity/trivy/issues/3007

File: internal/adapters/cloudformation/aws/elb/loadbalancer.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ func getListeners(lbr *parser.Resource, ctx parser.FileContext) (listeners []elb
 			listener := elb.Listener{
 				Metadata:       r.Metadata(),
 				Protocol:       r.GetStringProperty("Protocol", "HTTP"),
-				TLSPolicy:      r.GetStringProperty("SslPolicy", "ELBSecurityPolicy-2016-08"),
+				TLSPolicy:      r.GetStringProperty("SslPolicy", ""),
```
Comment (simar7) [inline internal/adapters/cloudformation/aws/elb/loadbalancer.go:36]: Is there a failing test that we would need to update after this change? If not, we should add a test for it.

Comment (SanaaYousaf) [reply internal/adapters/cloudformation/aws/elb/loadbalancer.go:36]: No, there is no any test file for cloudformation adapter.

Comment (simar7) [reply internal/adapters/cloudformation/aws/elb/loadbalancer.go:36]: OK can we add a simple unit test in that case? The fix is straight forward but having a unit test will guard us against regressions in the future.

Comment (SanaaYousaf) [reply internal/adapters/cloudformation/aws/elb/loadbalancer.go:36]: I did not write unit test for cloudformation file before, so i have no idea how to test it..

Comment (simar7) [reply internal/adapters/cloudformation/aws/elb/loadbalancer.go:36]: OK looks like CloudFormation could be better tested. I'll open a separate issue for it as it seems there's no test in place at the moment. 

For your reference, we test the terraform equivalent as so: https://github.com/aquasecurity/defsec/blob/master/internal/adapters/terraform/aws/elb/adapt_test.go

Comment (simar7) [reply internal/adapters/cloudformation/aws/elb/loadbalancer.go:36]: Issue opened here: https://github.com/aquasecurity/defsec/issues/1257

**Review by simar7 — DISMISSED** [review]

**Review by simar7 — APPROVED** [review]

**Pull request merged**